app/Http/Controllers/Api/V1/CurrikiGo/LmsServicesController.php

```diff
@@ -44,6 +44,7 @@ public function __construct(LmsSettingRepositoryInterface $lmsSettingRepository,
     }
 
     /**
+     * Login to Canvas
```

Comment:
Add new line between title and description.

Reply:
Issue still persists.

Reply:
Done!